test/extensions/INTEL/SPV_INTEL_joint_matrix/joint_matrix_load_store_offset.ll

```diff
@@ -0,0 +1,152 @@
+; This is an adapted copy of test/extensions/INTEL/SPV_INTEL_joint_matrix/joint_matrix.ll
```

Comment:
Lets use KHR/SPV_KHR_cooperative_matrix test as the template...

Reply:
Thanks for the suggestions! I have updated the test :)